server/modules/surveyRdb/repository/dataView/create.js

```diff
@@ -38,6 +38,28 @@ const _getSelectFieldKeys = (viewDataNodeDef) => {
   return keys.length > 0 ? `${SQL.jsonBuildObject(...keys)} AS ${ViewDataNodeDef.columnSet.keys}` : ''
 }
 
+const _getGroupByColumns = (viewDataNodeDef) => {
+  const { tableData, viewDataParent } = viewDataNodeDef
+  const groupByColumns = []
+  if (NodeDef.isMultiple(viewDataNodeDef.nodeDef)) {
+    groupByColumns.push(tableData.columnId)
+    groupByColumns.push(..._getSelectFieldNodeDefs(viewDataNodeDef).filter((s) => !s.includes(' AS ')))
+  }
+
+  //  get parent keys names to groupby
+  if (viewDataParent) {
+    const keys = Survey.getNodeDefKeys(viewDataParent.nodeDef)(viewDataParent.survey)
+      .map((nodeDef) => {
+        const columnNodeDef = new ColumnNodeDef(viewDataParent, nodeDef)
+        return `${viewDataParent.alias}.${columnNodeDef.name}`
+      })
+      .flat()
+
+    groupByColumns.push(...(keys || []))
```

Comment:
I think it's enough to add this to the group by generated before, without adding all the select fields that are not aliased, like in line 46